src/components/Login/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,41 @@
+import { Box, TextField, Button } from "@mui/material";
+import React from "react";
+
+interface LoginFormProps {
+  handleClick: () => Promise<void>
```

Comment:
why handle click returns Promise? And where promise is handled?

Reply:
fixed